ui/app/helpers/utils/conversions.util.test.js

```diff
@@ -1,6 +1,18 @@
 import assert from 'assert'
+import { ETH } from '../constants/common'
 import * as utils from './conversions.util'
 
+describe('getWeiHexFromDecimalValue', function () {
+  it('should correctly convert 0 in ETH', function () {
+    const weiValue = utils.getWeiHexFromDecimalValue({
+      value: '0',
+      fromCurrency: ETH,
+      fromDenomination: ETH,
+    })
+    assert.equal(weiValue, '0')
+  })
+})
+
```

Comment:
Trivial test case but writing it was the easiest way to confirm formatting so why not add it?